### DISCUSSION TITLE
tech/2023/03/why-kde

### DISCUSSION THREAD

**Discussion by giscus** [shikhalev/shikhalev.github.io#16] [General]
# 2023/03/why-kde

Некоторые размышления об окружении рабочего стола

https://shikhalev.github.io/tech/2023/03/why-kde.html

**Comment by alexeystalker** [comment]
Btw, "мозаичный режим" выглядит точь-в-точь как FancyZones для Шиндовс, представленный в 2019 году. Интересно, когда его начали пилить...

**Reply by shikhalev** [reply]
А черт его знает. [В истории коммитов он сразу внесен почти готовым](https://github.com/KDE/kwin/commit/e4507861f7b5ba563ae7cd5ce036a42a09f9edf8). И там, кстати, так и написано:
> * UI and behavior is a bit similar to the Windows Fancy Zones addon: https://docs.microsoft.com/en-us/windows/powertoys/fancyzones
